Lionel work ex 4 (#22)
mitre/fhir-exercises: Exercise_4/docs/index.md

# Welcome!

## Overview

> This exercise is intended to augment [FHIR for Research Exercises 0-3](https://github.com/mitre/fhir-exercises) with a broad view of EHR, terminology systems, and how USCDI and Implementation Guides like US Core enable remote access and interoperability.

Reviewer: Suggested change:

```suggestion
This exercise is intended to augment the R and Jupyter notebooks within the [FHIR for Research Exercises](https://github.com/mitre/fhir-exercises) course with a broad view of Electronic Health Record (EHR), terminology systems, and how United States Core Data for Interoperability (USCDI) and Implementation Guides like US Core enable remote access and interoperability.
```

## Sections:

### Part 1: EHR Data

> **Background:**  Electronic Health Records (EHR) contain important healthcare-related data. Understanding the purpose and structure of data inside the EHR is a necessary foundation for understanding how to access and exchange it.

Reviewer 2: ```suggestion
**Background:**  Electronic Health Record (EHRs) contain important healthcare-related data. Understanding the purpose and structure of data inside an EHR is a necessary foundation for understanding how to access and exchange it.
```

> **Background:**  Electronic Health Records (EHR) contain important healthcare-related data. Understanding the purpose and structure of data inside the EHR is a necessary foundation for understanding how to access and exchange it.

Reviewer: ```suggestion
**Background:**  Electronic Health Records (EHR) contain important healthcare-related data. Understanding the purpose and structure of data inside the EHR is a necessary foundation for understanding what data is captured, how it is captured, and how it might be queryable through an API.
````

> **Learning Objectives:** for NIH Researchers or Program Officers to be able to identify how and why EHR data can be used for research purposes.

Reviewer 2: Should this be NIH specific

```suggestion
**Learning Objectives:** Be able to identify how and why EHR data can be used for research purposes.
```

**Components:**

* Background
* Data Composition
  * What information is contained in an EHR record
  * Patient flow and associated data inuts
  * Codes and medical terminology

>   * File-types

Reviewer 2: This needs to be updated (e.g. I don't think file-types are talked about in Part 1 anymore). This should either give the high level topics or repeat the table of cotents of each section, at least for the top level. These could also be links directly to the relevant section.

```suggestion
**Components:**

* Overview of the EHR Technology Ecosystem
* Access and Regulatory Considerations
* Data Composition
* Codes and Medical Terminology
```

### Part 2: USCDI and Associated Terminology

> **Background:** Understanding the origin of the EHR data that researchers can work with is the first step to understanding how to eventually access and utilize it. A next step is understanding the technical standards that enable remote access and interoperability.

Reviewer 2: This seems more like a justification than a background/summary.

> **Background:** Understanding the origin of the EHR data that researchers can work with is the first step to understanding how to eventually access and utilize it. A next step is understanding the technical standards that enable remote access and interoperability.

Reviewer: ```suggestion
**Background:** Understanding EHR data that researchers could possibly work with is the first step to understanding how to eventually access and utilize it. A next step is understanding the technical standards that enable remote access and interoperability.
```
That might not be able to for lots of reasons.

> **Learning Objective:** o introduce NIH Program Officers and Researchers to the concept of health data exchange and interoperability, and an understanding of the core interoperable standard United States Core Data for Interoperability (USCDI). 

Reviewer 2: ```suggestion
**Learning Objective:** Understand the concept of health data exchange and interoperability, and the core interoperable standard United States Core Data for Interoperability (USCDI). 
```

> **Learning Objective:** o introduce NIH Program Officers and Researchers to the concept of health data exchange and interoperability, and an understanding of the core interoperable standard United States Core Data for Interoperability (USCDI). 

Reviewer: ```suggestion
**Learning Objective:** Understand the concept of health data exchange, interoperability, and the United States Core Data for Interoperability (USCDI) standard. 
```

**Components:**

* Background
  * Goals of healthcare data interoperability

>   * Universal data standards and mapping approach

Reviewer: ```suggestion
  * Data standards and mapping
```
It is not universal.

* USCDI
  * Background and history
  * Structure of Standard
  * Classes and Component Elements
  * Mapping USCDI to EHR data components

> * USCDI and FHIR

Reviewer 2: Update to reflect current table of contents.

Reviewer: Agree.

### Part 3: Implementation Guides and US Core

> **Background:** At this stage, you are now familiar with EHR data, and its conversion to standardized elements through a standard like USCDI. But while USCDI provides a standard for **WHAT** information must be included in an interoperable framework, it doesn’t specify **HOW** it must be packaged and rendered to external entities. The final piece in the interoperable framework then, is a standard geared precisely for information exchange. 

Reviewer: ```suggestion
**Background:** By Part 3, readers should be familiar with EHR data, and its conversion to standardized elements through a standard like USCDI. But while USCDI provides a standard for **WHAT** information must be included in an interoperable framework, it doesn’t specify **HOW** it must be packaged and rendered to external entities. The final piece in the interoperable framework then, is a standard geared precisely for information exchange. 
```

> This standard is known as FHIR®, or the Fast Healthcare Interoperability Resources. (See:  [hl7.org/fhir](hl7.org/fhir))

Reviewer: ```suggestion
This standard is known as HL7 FHIR®, or the Fast Healthcare Interoperability Resources. (See:  [hl7.org/fhir](https://hl7.org/fhir))
```

> **Learning Objective:** To help NIH Program Officers and Researchers understand the role that Implementation Guides like US Core play in standardizing the precise data elements made available in specific FHIR-enabled data exchanges. 

Reviewer 2: See above comments about this being NIH specific.

> **Learning Objective:** To help NIH Program Officers and Researchers understand the role that Implementation Guides like US Core play in standardizing the precise data elements made available in specific FHIR-enabled data exchanges. 

Reviewer: ```suggestion
**Learning Objective:** Understand the role that Implementation Guides like US Core play in standardizing the precise data elements made available in specific HL7 FHIR-enabled data exchanges. 
```
